src/pubsub/ua_pubsub_keystorage.h

```diff
@@ -112,6 +115,17 @@ typedef struct UA_PubSubKeyListItem {
 /* Queue Definition*/
 typedef TAILQ_HEAD(keyListItems, UA_PubSubKeyListItem) keyListItems;
 
+/**
+ * @brief It is used to hold configuration information required to connect an SKS server
+ * and fetch the security keys
+ */
+typedef struct UA_PubSubSKSConfig {
```

Comment:
No need to prefix the members with sks.
The structure is already prefixed.

I do prefer to have the clientconfig as a direct member, not a pointer.
Less moving parts. Less malloc error handling.